src/colorizer/ColorizeCanvas3D.ts

```diff
@@ -273,8 +303,15 @@ export class ColorizeCanvas3D implements IRenderCanvas {
   }
 
   getIdAtPixel(x: number, y: number): number {
-    if (this.volume?.isLoaded()) {
-      return this.view3d.hitTest(x, y) - 1;
+    const dataset = this.params?.dataset;
+    // TODO: Currently `View3d.hitTest` reports the per-frame IDs, not global IDs.
+    // Ideally, vole-core should handle this based on the frame ID offset array that's passed into it
+    // during colorizer setup.
```

Comment:
Things may have evolved already, but I slightly disagree with the comment here.  I think that vole-core should be reporting out the id that was clicked on in the pick buffer, and any adjustment is up to the application (TFE in this case, but not necessarily TFE, think simularium or CFE etc etc)

Reply:
Will remove the TODO. I agree that this sounds reasonable, and I also do this operation in the 2D canvas in #645